source/core/GenericFunctions.cpp

```diff
@@ -2676,6 +2686,9 @@ DEFINE_VIREO_BEGIN(Generics)
     DEFINE_VIREO_FUNCTION(CopyEnum4, "p(i(UInt32)  o(UInt32) i(UInt32))");
     DEFINE_VIREO_FUNCTION(CopyEnum8, "p(i(UInt64)  o(UInt64) i(UInt64))");
 
+    // Instruction to copy boolean value and reset it to another value
+    DEFINE_VIREO_FUNCTION(CopyAndReset, "p(io(Boolean from)  o(Boolean to) i(Boolean fromNewValue))");
```

Comment:
'fromNewValue' reads strangely.   I'd suggest renaming params to source, dest, and sourceNewValue or sourceResetValue.

Reply:
Wow. Perfect coincidence on the comment :-)